### PR TITLE
Feature configure extra fields [#151650459]

### PR DESCRIPTION
**Pull request by the-jedi-droid**
[#151650459]
- Adds new APIs to Configuration builder.
- Adds a new class to collect extra fields. 

Please review :)

### CODE REVIEW

File: example/src/main/java/com/openlocate/example/adapters/SafeGraphLocationListAdapter.java [hunk 1]
```diff
@@ -28,15 +28,15 @@
 import android.widget.TextView;
 
 import com.openlocate.example.R;
-import com.openlocate.example.models.SafeGraphPlace;
+import com.openlocate.example.models.GooglePlace;
 
 import java.util.List;
 
 public class SafeGraphLocationListAdapter extends RecyclerView.Adapter<SafeGraphLocationListAdapter.SafeGraphLocationViewHolder> {
```
Comment (andrewisrael) [inline example/src/main/java/com/openlocate/example/adapters/SafeGraphLocationListAdapter.java]: Can you update the classname too?

File: example/src/main/java/com/openlocate/example/network/GooglePlacesApiClient.java [hunk 2]
```diff
@@ -22,17 +22,17 @@
...
 import java.util.Map;
 
 import retrofit2.Call;
 import retrofit2.http.GET;
 import retrofit2.http.QueryMap;
 
-public interface SafeGraphPlaceClient {
+public interface GooglePlacesApiClient {
 
     @GET("places/v1/nearby")
```
Comment (andrewisrael) [inline example/src/main/java/com/openlocate/example/network/GooglePlacesApiClient.java]: Safegraph and google have the same places api url?

File: openlocate/src/main/java/com/openlocate/android/core/LocationProvider.java [hunk 3]
```diff
@@ -47,9 +49,9 @@
         this.value = value;
     }
 
-   String getValue() {
+    String getValue() {
```
Comment (andrewisrael) [inline openlocate/src/main/java/com/openlocate/android/core/LocationProvider.java]: Could you fix the formatting?

**Comment by the-jedi-droid** [comment]
Please review.

**Pull request merged**